includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -2453,6 +2459,48 @@ private function remove_disallowed_descendants( DOMElement $node, $allowed_desce
 		}
 	}
 
+	/**
+	 * Loop through node's siblings and remove them.
+	 *
+	 * @param DOMElement $node      Node.
+	 * @param string     $spec_name Spec name.
+	 */
+	private function remove_disallowed_siblings( DOMElement $node, $spec_name ) {
+		if ( ! $node->previousSibling || ! $node->nextSibling ) {
+			return;
+		}
+
+		$prev_sibling = $node->previousSibling;
+		while ( null !== $prev_sibling ) {
+			if ( XML_COMMENT_NODE !== $prev_sibling->nodeType ) {
+				$this->remove_invalid_child(
+					$prev_sibling,
+					[
+						'code'      => self::DISALLOWED_SIBLING_TAG,
+						'sibling'   => $node->nodeName,
+						'spec_name' => $spec_name,
+					]
+				);
+			}
+			$prev_sibling = $prev_sibling->previousSibling;
```

Comment:
Humm. If the `$prev_sibling` was just removed, would its `$prev_sibling->previousSibling` always still be set? Should this value be captured before the `if` statement just in case? Same goes for the below statement.

Reply:
It seems to work, but sure I can capture it before 👍  Might make it more readable too